src/main/java/io/neow3j/examples/contractinvoke/EscrowExampleCreateTrustAgreement.java

```diff
@@ -0,0 +1,169 @@
+package io.neow3j.examples.contractinvoke;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import io.neow3j.contract.GasToken;
+import io.neow3j.contract.SmartContract;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.response.InvocationResult;
+import io.neow3j.protocol.core.response.NeoSendRawTransaction;
+import io.neow3j.protocol.core.stackitem.StackItem;
+import io.neow3j.protocol.http.HttpService;
+import io.neow3j.transaction.AccountSigner;
+import io.neow3j.types.ContractParameter;
+import io.neow3j.types.Hash160;
+import io.neow3j.wallet.Account;
+
+public class EscrowExampleCreateTrustAgreement {
+        public static void main(String[] args) throws Throwable {
+                /*
+                 * This example will create a Trust agreement between a trustor, an arbiter and
+                 * a beneficiary.
+                 * Initially, you will run the `createAgreement` function to create a Trust
+                 * agreement where you will specifiy who the arbiter and beneficiary is. You
+                 * will automatically be assigned as the trustor.
+                 * 
+                 * Then you will run the `getAgreement` function to get the details of the
+                 * agreement you just created so you can see the agreement.
+                 * 
+                 * Next you will run the `executeAgreement` function provided in the
+                 * `EscrowExampleExecuteTrustAgreemnt` class file to execute the agreement as
+                 * the arbiter. This will transfer the funds to the beneficiary.
+                 * 
+                 * In between the last 2 steps, you can run the `getContractBalance` utility
+                 * function to get the total GAS balance of the Escrow to check if your code
+                 * worked.
```

Comment:
```suggestion
                 * This example will create a Trust agreement between a trustor, an arbiter and
                 * a beneficiary.
                 * 
                 * Initially, you will run the `createAgreement` function to create a Trust
                 * agreement where you will specify who the arbiter and beneficiary is. You
                 * will automatically be assigned as the trustor.
                 * 
                 * Then you will run the `getAgreement` function to get the details of the
                 * agreement you just created so you can see the agreement.
                 * 
                 * Next you will run the `executeAgreement` function provided in the
                 * `EscrowExampleExecuteTrustAgreemnt` class file to execute the agreement as
                 * the arbiter. This will transfer the funds held in the Escrow contract
                 * to the beneficiary.
                 * 
                 * In between the last 2 steps, you can run the `getContractBalance` utility
                 * function to get the total GAS balance of the Escrow to check if your code
                 * worked and the Escrow contract actually holds the balance you specified
                 * in the agreement.
```